### PR TITLE
Setup script: Restart pow if exists; verbose output

### PR DESCRIPTION
**Pull request by olleolleolle**
This PR tries to slightly improve the setup script for new developers.

  - make the setup script output mention its name in echo'ed messages
  - ~add a conditional around touching a tmp/restart.txt file~ try to create the `tmp/` directory, to avoid failures for those who are not using the pow system

### CODE REVIEW

File: script/setup [hunk 1]
```diff
@@ -9,31 +9,33 @@ else
...
-  echo "Reloading the database"
+  echo "$0: Reloading the database"
   { rake db:create:all db:drop:all db:setup --trace
     rake db:test:prepare --trace
   } >&3 2>&1
 fi
 
-echo "Restarting Pow/Passenger"
-touch tmp/restart.txt
+if [ -d tmp ]; then
```
Comment (dwradcliffe) [inline script/setup]: I think tmp is used for other things as well? Maybe better to create the directory here if it doesn't exist? I don't want to couple this to pow but seems ok to touch this file.

Comment (simi) [inline script/setup]: We can also add `tmp/.keep` file to ensure this directory exist.

Comment (dwradcliffe) [inline script/setup]: Yeah I'm ok with that too

**Comment by olleolleolle** [comment]
(I made it a local `mkdir -p` instead - this avoids having to mess with `.gitignore` rules, which include `tmp/`.)

**Comment by olleolleolle** [comment]
@dwradcliffe Thanks for the review!

**Comment by olleolleolle** [comment]
@dwradcliffe Oh, would this change now be acceptable?

**Comment by dwradcliffe** [comment]
Thanks!

**Pull request merged**